templates/fastcgi.conf

```diff
@@ -25,6 +26,9 @@ fastcgi_param QUERY_STRING $query_string;
 fastcgi_param SCRIPT_NAME $fastcgi_script_name;
 fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
 
+fastcgi_param ADD_X_FORWARDED_FOR $proxy_add_x_forwarded_for;
+fastcgi_param REQUEST_ID $request_id;
```

Comment:
i would call this one:

`nginx_request_id` or  `webserver_request_id`

Reply:
changed